images.yaml

```diff
@@ -374,6 +374,24 @@
     - tagSuffix: giantswarm
       dockerfileOptions:
       - RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
+- name: k8s.gcr.io/kube-apiserver
+  tagTrimVersionPrefix: true
```

Comment:
Added to transform `v1.20.0` into `1.20.0` to match https://github.com/giantswarm/k8scloudconfig/blob/cbfb99e4baec60525469f8b262fffc86dddad7e4/pkg/template/release.go#L19. Alternatively, we would need a new major release of `k8scloudconfig` that doesn't strip the "v" prefix I think.